src/containers/ProfileDetails/components/RF/index.js

```diff
@@ -0,0 +1,459 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Form, Input, Select, Tooltip  } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import styles from '../index.module.scss';
+import { RADIOS } from '../../constants';
+
+const { Item  } = Form;
+const { Option } = Select;
+
+const RFForm = ({ form, details}) => {
+
+    useEffect(() => {
+        const formData = {
+            rfConfigMap: {},
+        };
+        const currentRadios = details.rfConfigMap ? Object.keys(details.rfConfigMap) : RADIOS;
```

Comment:
this ternary operator is never false because of the defaultProp, just put `Object.keys(details.rfConfigMap)`